src/App.tsx

```diff
@@ -2,104 +2,24 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+
+const preparedPosts = postsFromServer.map(post => (
+  {
+    ...post,
+    poster: usersFromServer.find(user => user.id === post.userId) || null,
+    comments: commentsFromServer.filter(
+      comment => comment.postId === post.id,
+    ) || null,
```

Comment:
If you filter -> you don't need to assign null by default, cause filter returns [] empty array